code/modules/food&drinks/food/snacks_burgers.dm

```diff
@@ -18,6 +18,26 @@
 	desc = "A bloody burger."
 	bonus_reagents = list("vitamin" = 4)
 
+/obj/item/weapon/reagent_containers/food/snacks/burger/human/lizard
+	name = "lizard burger"
+	desc = "Made from non-valid tailmeat."
+	bonus_reagents = list("vitamin" = 4)
+
+/obj/item/weapon/reagent_containers/food/snacks/burger/human/bird
+	name = "bird burger"
+	desc = "Tastes like chicken."
+	bonus_reagents = list("vitamin" = 4)
+
+/obj/item/weapon/reagent_containers/food/snacks/burger/human/robo
```

Comment:
Yeah you should probably make this something different, like a metal burger or a screen burger or something that isn't already defined.

Stay away from the roburger.